subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -157,6 +157,16 @@ config BT_CTLR_ADV_SYNC_SET
 	help
 	  Maximum supported periodic advertising sets.
 
+config BT_CTLR_ADV_DATA_BUF_MAX
+	int "AD Data Max. Buffers"
```

Comment:
I think it would be better to avoid abbreviations here, i.e. AD => Advertising Data and Max. => Maximum